### PR TITLE
DEV-88 Lista placówek RCKiK

### PR DESCRIPTION
**Pull request by adjurk**
- Adds BloodCenterView, along with BloodCenterDetail and a MapView
  - All views currently fetch sample data, but models were created with the API mappings in mind.
  - MapView isn't centered to the building yet
  - BloodCenterBankDetails shows the same sample data for all centres
  
Next step: merging this with @Łukasz's networking groundwork to pull data from the API 🚀

### CODE REVIEW

File: BloodDonor/Modules/BloodCenter/BloodCenter.swift [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+//            switch response.result {
+//            case .success(let data):
+//                completion(response.data)
+//
+//            case .failure(let error):
+//                completion(.failure(error))
+//
+//            default:
+//                fatalError()
+//            }
```
Comment (s-kakol) [inline BloodDonor/Modules/BloodCenter/BloodCenter.swift]: Unless these are left for a purpose I'd remove comments

File: BloodDonor/Modules/BloodCenter/BloodCenter.swift [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+}
+
+class BloodCenterDataService {
+    var apiBaseUrl =  "https://api.pre.gwn.adamjurkiewicz.com"
+    let apiToken: String
+    let headers: HTTPHeaders
+    
+    init() {
+        apiToken = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJibG9vZGRvbm9yIiwiY29udGV4dCI6eyJ1c2VyIjp7InVzZXJJZCI6MSwiZGlzcGxheU5hbWUiOiJhZGFtaiJ9fSwiaWF0IjoxNjg1NTUwNjI1LCJleHAiOjE2ODYxNTU0MjV9.eauTNijJm2fXBjvKZFY0A3tqdLTUwKnk0jcGqY7aqEg"
```
Comment (s-kakol) [inline BloodDonor/Modules/BloodCenter/BloodCenter.swift:40]: This is temporary solution, yes?

**Review by s-kakol — COMMENTED** [review]
Skipping some of the commented lines and to the best of my limited knowledge of swift - LGTM
If you could respond to the comments I'll go ahead and approve.

File: BloodDonor/Modules/BloodCenter/BloodCenter.swift (hunk 2 above)
Comment (adjurk) [reply BloodDonor/Modules/BloodCenter/BloodCenter.swift:40]: Yep - this allowed me to simulate how would Swift map the JSON that it receives from the API (and it did well in that regard!).

File: BloodDonor/Modules/BloodCenter/BloodCenter.swift (hunk 1 above)
Comment (adjurk) [reply BloodDonor/Modules/BloodCenter/BloodCenter.swift]: These were my (unfortunately unsuccessful) tests of completion handlers for asynchronous data fetching with the Alamofire library. I've removed them now 👍

**Review by s-kakol — APPROVED** [review]

**Pull request merged**